backend/api/Database/Models/InspectionArea.cs

```diff
@@ -19,5 +19,7 @@ public class InspectionArea
         [Required]
         [MaxLength(200)]
         public string Name { get; set; }
+
+        public string? AreaPolygonJson { get; set; } = "";
```

Comment:
Why have it as a string here and not just use the InspectionAreaPolygon datatype?